backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -128,33 +77,93 @@ private static Message createMessageWithEmail(MimeMessage emailContent)
    * @return the MimeMessage to be used to send email
    * @throws MessagingException if there was a problem accessing the Store
    */
-  private MimeMessage createEmail(String subject, String bodyText) throws MessagingException {
+  private MimeMessage createEmailWithSingleRecipient(
+      String recipientUsername, String subject, String bodyText) throws MessagingException {
 
     Properties props = new Properties();
     Session session = Session.getDefaultInstance(props, /* authenticator= */ null);
 
     MimeMessage email = new MimeMessage(session);
 
     email.setFrom(new InternetAddress(APPLICATION_EMAIL));
-    email.addRecipient(RecipientType.TO, new InternetAddress(recipientEmail));
+    email.addRecipient(RecipientType.TO, new InternetAddress(recipientUsername + "@google.com"));
     email.setSubject(subject);
     email.setText(bodyText);
     return email;
   }
 
+  /**
+   * Create a MimeMessage using the parameters provided.
+   *
+   * @param bodyText body text of the email
+   * @return the MimeMessage to be used to send email
+   * @throws MessagingException if there was a problem accessing the Store
+   */
+  private MimeMessage createEmailWithMultiRecipients(
+      String firstRecipientUsername, String secondRecipientUsername, String bodyText)
+      throws MessagingException {
+
+    Properties props = new Properties();
+    Session session = Session.getDefaultInstance(props, /* authenticator= */ null);
+
+    MimeMessage email = new MimeMessage(session);
+
+    email.setFrom(new InternetAddress(APPLICATION_EMAIL));
+    email.addRecipients(
+        RecipientType.TO,
+        new InternetAddress[] {
+          new InternetAddress(firstRecipientUsername + "@google.com"),
+          new InternetAddress(secondRecipientUsername + "@google.com")
+        });
+    email.setSubject("Ad-lib: Match Found");
+    email.setText(bodyText);
+    return email;
+  }
```

Comment:
No need for email to multiple recipients because each user in the match will get notifications at different times! (every 5 seconds)

Reply:
Ack

Reply:
Deleted Function